### PR TITLE
Add support for batch events

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: lib/pusher/client.rb [hunk 1]
```diff
@@ -269,13 +269,35 @@ def trigger(channels, event_name, data, params = {})
       post('/events', trigger_params(channels, event_name, data, params))
     end
 
+    # Trigger multiple events at the same time
+    #
+    # POST /apps/[app_id]/batch_events
+    #
+    # @param events [Array] List of events to publish
```
Comment (Nicowcow) [inline lib/pusher/client.rb]: worth documenting the use `trigger_batch(e1, e2, ...)` ?

**Comment by Nicowcow** [comment]
Generally, same comment as https://github.com/pusher/pusher-http-go/pull/23#discussion_r64016641 for consistency between functions

File: lib/pusher/client.rb (hunk 1 above)
Comment (zimbatm) [inline lib/pusher/client.rb]: i think it's good enough, i don't know if `@param` supports variadic arguments. either case it works the same so it's not a biggie if the user misses it

**Comment by jpatel531** [comment]
👍

**Comment by zimbatm** [comment]
It's a valid point but standardization would only work if backward-compatibility is broken. I'm not prepared to do that in this release.

**Pull request merged**